sklearn/tests/test_naive_bayes.py

```diff
@@ -438,6 +445,45 @@ def test_discretenb_coef_intercept_shape(cls):
     assert clf.intercept_.shape == (1,)
 
 
+@pytest.mark.parametrize('cls', [MultinomialNB, ComplementNB, BernoulliNB,
+                                 CategoricalNB])
+@pytest.mark.parametrize('use_partial_fit', [False, True])
+@pytest.mark.parametrize('num_samples,expected_first_axis_length',
+                         [(3, 2), (2, 1)])
+def test_discretenb_degenerate_one_class_case(
+        cls,
```

Comment:
```suggestion
        NaiveBayes,
```

Reply:
To make this change, I think we'd also need to change line 448, so I will do this locally.

Reply:
Done in 4c5f8d2